### PR TITLE
Update Introduction for User Guide

### LINKED ISSUE #334: Update UG Introduction

### PR DESCRIPTION
**Pull request by LinkesV**
Fixes #334

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1,37 +1,66 @@
 
-# Vendor Vault User Guide
+# VendorVault User Guide
 
-Vendor Vault is a **desktop app for managing supplier contact information and deliveries, optimized for use via a  Line Interface** (CLI). If you can type fast, VendorVault can get your contact management tasks done faster than traditional GUI apps. Vendor Vault is targeted at small convenience/grocery stores. 
+VendorVault is a **desktop app for managing supplier contact information and deliveries, optimized for use via a Command Line Interface** (CLI). If you can type fast, VendorVault can get your contact management tasks done faster than traditional GUI apps. Vendor Vault is targeted at small convenience/grocery stores.
```
Comment (CJianzhi) [inline docs/UserGuide.md:4]: Should there be a spacing between Vendor Vault on the last line/

File: docs/UserGuide.md [hunk 2]
```diff
@@ -1,37 +1,66 @@
...
+
+
+---
 
 --------------------------------------------------------------------------------------------------------------------
 
-## Quick start
+## How to get started
+
+1. If you are new to Vendor Vault, we suggest starting from the [Setting up VendorVault section](#setting-up-vendorvault).
```
Comment (CJianzhi) [inline docs/UserGuide.md:17]: Spacing between VendorVault

File: docs/UserGuide.md [hunk 3]
```diff
@@ -1,37 +1,66 @@
...
+1. Java is the toolset that will allow you to use VendorVault on your system. Hence, first you need to ensure that you have Java `17` or above installed on your Computer. If you are unsure, you can check by following the steps below:
+   - 1a. Open up your command terminal. 
+     * Windows users can open the terminal by typing in `Command Prompt` in the search bar located at the bottom left of the screen. 
+     * Mac users can open the terminal by typing in `Terminal` using the spotlight search bar (accessed by pressing Command + Space).
+     * Linux users can open for the command terminal by searching for the `Terminal` or `Console` in the Applications Menu
+   
+   - 1b. Type in java -version to check if you have Java installed. If you have java installed you should see something like this:
+     ![Terminal Example showcasing java version](images/intro/terminalJavaVersion.png)
+   
+   - 1c. If you do not have Java installed, you can download it from [here](https://www.oracle.com/java/technologies/downloads/#java17?er=221886).
```
Comment (CJianzhi) [inline docs/UserGuide.md:36]: Should we specify for mac users they need the specific AZUL 17 distribution?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -1,37 +1,66 @@
...
+![Ui](images/intro/Ui.png)
+
+5. Type commands in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
 
-   * `list` : Lists all contacts.
+   * `list -a` : Lists all suppliers and deliveries.
 
-   * `add -s n/John Doe p/98765432 e/johnd@example.com com/John street, block 123, #01-01 pro/ iPhone` : Adds a supplier named `John Doe` to the VendorVault.
+   * `add -s n/John Doe p/98765432 e/johnd@example.com com/John street, block 123, #01-01 pro/iPhone` : Adds a supplier named `John Doe` to the VendorVault.
```
Comment (CJianzhi) [inline docs/UserGuide.md:57]: Should "the" be removed ?

File: docs/UserGuide.md [hunk 5]
```diff
@@ -120,7 +110,6 @@ Lists all suppliers and deliveries in the VendorVault
 Format: `list -a`
 
 <box type="warning" seamless>
-
 **Warnings**:
 - No other parameters should be given for this command.
 - At least one space between list and -a
```
Comment (CJianzhi) [inline docs/UserGuide.md:115]: Will need to update this to be same as Jian yi's one later

File: docs/UserGuide.md [hunk 6]
```diff
@@ -526,5 +515,40 @@ _Details coming soon ..._
...
+
+### Supplier-specific Commands
+
+Action     | Format, Examples
+-----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------
+**[Add](#adding-a-supplier-add-s)**    | `add -s n/NAME p/PHONE e/EMAIL com/COMPANY [t/TAG]…​ [pro/PRODUCT]…​` <br> e.g., `add -s n/John Doe p/98765432 e/johnd@example.com com/companyA t/friends t/owesMoney pro/rice pro/bread`
+**[Delete](#deleting-a-supplier-delete-s)** | `delete -s INDEX`<br> e.g., `delete -s 3`
+**[List](#listing-all-suppliers-list-s)**   | `list -s`
+**[Mark](#mark-a-supplier-with-a-status-mark-s)**   | `mark -s INDEX STATUS`<br> e.g.,`mark -s 2 active`
+**[Find](#find-a-supplier-find-s)**   | `find -s n/<KEYWORD FOR SUPPLIER NAME> com/<KEYWORD FOR SUPPLIER COMPANY> pro/<KEYWORD FOR SUPPLIER PRODUCT>` <br> e.g., `find -s n/link com/NU`
```
Comment (CJianzhi) [inline docs/UserGuide.md:537]: Should <KEYWORD FOR SUPPLIER NAME> be changed to the parameter name instead for standardisation

File: docs/UserGuide.md [hunk 7]
```diff
@@ -526,5 +515,40 @@ _Details coming soon ..._
...
+**[Sort](#sort-suppliers-sort-s)**   | `sort -s so/SORT_ORDER sb/SORT_BY_FIELD`<br> e.g., `sort -s so/a sb/n`
+
+
+### Delivery-specific Commands
+
+Action     | Format, Examples
+-----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------
+**[Add](#adding-a-delivery-add-d)**    | `add -d on/DELIVERY_DATE_TIME s/SUPPLIER_INDEX pro/PRODUCT q/QUANTITY kg/g/L/mL/units c/COST` <br> e.g., `add -d on/18-06-2024 17:00 s/1 pro/bread q/500 g c/5.50`
```
Comment (CJianzhi) [inline docs/UserGuide.md:545]: Need to remove the units behind QUANTITY as well

File: docs/index.md [hunk 8]
```diff
@@ -8,7 +8,7 @@
 [![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
 [![codecov](https://codecov.io/gh/se-edu/addressbook-level3/branch/master/graph/badge.svg)](https://codecov.io/gh/se-edu/addressbook-level3)
 
-![Ui](images/Ui.png)
+![Ui](images/intro/Ui.png)
 
 **AddressBook is a desktop application for managing your contact details.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
```
Comment (CJianzhi) [inline docs/index.md:13]: should we change AddressBook  to VV as well?

File: docs/UserGuide.md [hunk 9]
```diff
@@ -526,5 +515,40 @@ _Details coming soon ..._
...
+**[Delete](#deleting-a-delivery-delete-d)** | `delete -d INDEX`<br> e.g., `delete -d 3`
+**[List](#listing-all-deliveries-list-d)**   | `list -d`
+**[Mark](#marking-a-delivery-mark-d)**   | `mark -d INDEX STATUS`<br> e.g.,`mark -d 2 PENDING`
+**[Find](#find-a-delivery-find-d)**   | `find -d on/DELIVERY_DATE_TIME stat/STATUS s/SUPPLIER_INDEX pro/PRODUCT`<br> e.g., `find -d on/ 28-06-2025 17:00 pro/ milk`
+**[Sort](#sort-deliveries-sort-d)**   | `sort -d so/SORT_ORDER sb/SORT_BY_FIELD`<br> e.g., `sort -d so/a sb/c`
+**[Upcoming](#upcoming-deliveries-upcoming)** | `upcoming aft/START_DATE bef/END_DATE`<br> e.g., `upcoming aft/19-12-2022 08:00 bef/18-06-2023 17:00`
```
Comment (CJianzhi) [inline docs/UserGuide.md:551]: Should we change START_DATE and END_DATE. to DELIVERY_DATE_TIME to be consistent with whats shown on the app?

**Review by CJianzhi — COMMENTED** [review]
LGTM! Some minor issues that needs to be discussed further on whether to change

File: docs/UserGuide.md (hunk 9 above)
Comment (LinkesV) [reply docs/UserGuide.md:551]: This section I will update after everyone has placed their command formats

File: docs/UserGuide.md (hunk 3 above)
Comment (LinkesV) [reply docs/UserGuide.md:36]: I dont this this this is needed correct?

File: docs/UserGuide.md (hunk 1 above)
Comment (ngjianyi) [reply docs/UserGuide.md]: Could the last line be more descriptive? Such as 'VendorVault specifically targets small convenience/grocery stores and aims to help them achieve higher efficiency!'

File: docs/UserGuide.md [hunk 10]
```diff
@@ -1,37 +1,66 @@
...
+
+--------------------------------------------------------------------------------------------------------------------
+
+## Setting up VendorVault 
+
+1. Java is the toolset that will allow you to use VendorVault on your system. Hence, first you need to ensure that you have Java `17` or above installed on your Computer. If you are unsure, you can check by following the steps below:
+   - 1a. Open up your command terminal. 
+     * Windows users can open the terminal by typing in `Command Prompt` in the search bar located at the bottom left of the screen. 
+     * Mac users can open the terminal by typing in `Terminal` using the spotlight search bar (accessed by pressing Command + Space).
+     * Linux users can open for the command terminal by searching for the `Terminal` or `Console` in the Applications Menu
```
Comment (ngjianyi) [inline docs/UserGuide.md:31]: Missing . at the end?

File: docs/UserGuide.md [hunk 11]
```diff
@@ -1,37 +1,66 @@
...
 --------------------------------------------------------------------------------------------------------------------
 
-## Quick start
+## How to get started
+
+1. If you are new to Vendor Vault, we suggest starting from the [Setting up VendorVault section](#setting-up-vendorvault).
+
+2. If you have already set up Vendor Vault and are looking for information of our commands and features, take a look at our [Features](#features) or [Command Summary](#command-summary) sections. 
+
+3. If you’re unsure about any of the terms used in this User Guide, please refer to the [Glossary](#glossary) section for definitions and explanations
```
Comment (ngjianyi) [inline docs/UserGuide.md:21]: Missing .?

**Review by ngjianyi — COMMENTED** [review]
LGTM, just some missing punctuation

**Pull request merged**